src/modules/avformat/consumer_avformat.c

```diff
@@ -1316,6 +1317,8 @@ static void *consumer_thread( void *arg )
 					img_fmt = mlt_image_rgb24a;
 				else if ( !strcmp( img_fmt_name, "yuv420p" ) )
 					img_fmt = mlt_image_yuv420p;
+				else if ( !strcmp( img_fmt_name, "yuv422p16" ) )
```

Comment:
Should this entire case statement just be replaced with the new mlt_image_format_id() function?